desks-web-app/src/components/booking-components/WeekOverviewWindow.vue

```diff
@@ -80,13 +111,9 @@ export default defineComponent({
       this.$emit("add-booking", booking)
     },
 
-    async mounted(){
-      await this.fetchBookings()
-    },
-
     async fetchBookings() {
       if (this.bookingDefaults && this.bookingDefaults.room) {
-        await this.fetchBookingsByRoomAndWeek({roomId: this.bookingDefaults.room.id, date: this.startOfWeek})
+        await this.fetchBookingsForRange({roomId: this.bookingDefaults.room.id, from: this.dateScope[0], to: this.dateScope[this.dateScope.length - 1]})
```

Comment:
The end of the last day should be included otherwise only the bookings of the logged in user will be displayed for the last day in range 
```suggestion
        await this.fetchBookingsForRange({
            roomId: this.bookingDefaults.room.id,
            from: this.dateScope[0],
            to: moment(this.dateScope[this.dateScope.length - 1]).endOf("day")
        })
```